File: 2022/other_languages/day01.py
import csv
a=csv.reader(open("input01.txt"))
input=[]
calories=[]
for i in a:
    if len(i)>0:
        input.append(int(i[0]))
    else:
        calories.append(sum(input))
        input=[]

calories.append(sum(input))

max(calories)
#69528

calories.sort()
sum(calories[-3:])
#206152

File: 2022/other_languages/day01.jl
open("input01.txt","r") do f
    global n = 0
    global calories = []
    while !eof(f)
        s = readline(f)
        if length(s)==0
            append!(calories,n)
            n = 0
        else
            n += parse(Int64,s)
        end
    end
end

append!(calories,n)

maximum(calories)
#69528

sum(sort(calories)[(end-2):end])
#206152